exe/sdr

```diff
@@ -108,7 +122,7 @@ end
 
 subcommands[command].order!
 
-options[:url] ||= 'https://sdr-api-prod.stanford.edu'
+options[:url] ||= 'https://sdr-api-qa.stanford.edu'
```

Comment:
We shouldn't change this. I want this to work for users in production without them having to specify the endpoint.

Reply:
@jcoyne How about if I put this in config, and those of us who don't want to inadvertently touch prod can override via local settings?